src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce(addStyle, {});
```

Comment:
Consider handling cases where `sourceString` might be an empty string or contain malformed style definitions. This could lead to unexpected behavior or errors when splitting and reducing the string.